### PR TITLE
Added Program to Check Anagram or Not 

### PR DESCRIPTION
**Pull request by Utsav1999**
### **Describe your change:**

This program will check whether two words are anagrams or not

* [X] Add an algorithm?
* [ ] Fix a bug or typo in an existing algorithm?
* [ ] Documentation change?

### **Checklist:**
* [X] I have read [CONTRIBUTING.md](https://github.com/TheAlgorithms/Python/blob/master/CONTRIBUTING.md).
* [X] This pull request is all my own work -- I have not plagiarized.
* [X] I know that pull requests will not be merged if they fail the automated tests.
* [ ] This PR only changes one algorithm file.  To ease review, please open separate PRs for separate algorithms.
* [X] All new Python files are placed inside an existing directory.
* [X] All filenames are in all lowercase characters with no spaces or dashes.
* [X] All functions and variable names follow Python naming conventions.
* [X] All function parameters and return values are annotated with Python [type hints](https://docs.python.org/3/library/typing.html).
* [ ] All functions have [doctests](https://docs.python.org/3/library/doctest.html) that pass the automated testing.
* [X] All new algorithms have a URL in its comments that points to Wikipedia or other similar explanation.
* [X] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Comment by Utsav1999** [comment]
Can someone please clarify me the below error?
`The command "pytest --doctest-modules --cov-report=term-missing:skip-covered --cov=. ." exited with 1.`

**Comment by cclauss** [comment]
https://travis-ci.com/github/TheAlgorithms/Python/builds/177843401#L797

Hint: `"listen"` should be a quoted string, not a variable name.

You need to rebase against master.  25 commits and modification of 3 different files is not going to work.

**Comment by Utsav1999** [comment]
I am new to Github. Can anyone please help me with how to rebase against master? My forked repo is showing that  **my brach is ahead of 31 commits than Algorithms/Python: master**

**Comment by spamegg1** [comment]
@Utsav1999 It may be easier to close this pull request, delete your forked repo, fork again, and make a new pull request. Try to make all your changes in one commit. Also you should modify only one algorithm file as explained in CONTRIBUTING.MD

**Comment by cclauss** [comment]
I use https://devguide.python.org/gitbootcamp/#syncing-with-upstream as my cheatsheet

File: maths/krishnamurthy_number.py [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+A Krishnamurthy Number is a number whose sum of the
+factorial of the digits equals to the original
+number itself.
+
+For example: 145 = 1! + 4! + 5!
+    So, 145 is a Krishnamurthy Number
+"""
+
+
+def factorial(digit: int) -> int:
```
Comment (deadshotsb) [inline maths/krishnamurthy_number.py:13]: @Utsav1999 Kindly also add docstrings to your function

Comment (cclauss) [reply maths/krishnamurthy_number.py:13]: Or `from math import factorial`

File: strings/anagram_check.py [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    """
+    >>> anagram("listen", "silent")
+    True
+    >>> anagram("rat", "bat")
+    False
+    >>> anagram("tool", "loot")
+    True
+    """
+
+    flag = 0
```
Comment (cclauss) [inline strings/anagram_check.py:22]: Flag should be boolean (True/False).

**Pull request closed without merging**